test/hack/resource-management/resourcetypes/instance.go

```diff
@@ -71,6 +71,39 @@ func (i *Instance) GetExpired(ctx context.Context, expirationTime time.Time) (id
 	return ids, err
 }
 
+func (i *Instance) GetCount(ctx context.Context) (count int, err error) {
+	var nextToken *string
+
+	for {
+		out, err := i.ec2Client.DescribeInstances(ctx, &ec2.DescribeInstancesInput{
+			Filters: []ec2types.Filter{
+				{
+					Name:   lo.ToPtr("instance-state-name"),
+					Values: []string{string(ec2types.InstanceStateNameRunning)},
+				},
```

Comment:
Do we want other instances too? What about Pending, Shutting-down, and stopped? 
https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-lifecycle.html